tree/30_generic_methods/file_content.cf

```diff
@@ -23,13 +23,13 @@
 # @parameter lines Line(s) to add in the file
 # @parameter enforce Enforce the file to contain only line(s) defined (true or false)
 # 
-# @class_prefix file_content
+# @class_prefix file_lines_present
```

Comment:
I don't think we should change the prefix of existing generic methods

Reply:
It was wrongly changed in the first rename PR.